src/modules/schnorrsig/main_impl.h

```diff
@@ -223,8 +225,14 @@ int secp256k1_schnorrsig_verify(const secp256k1_context* ctx, const unsigned cha
     secp256k1_gej_set_ge(&pkj, &pk);
     secp256k1_ecmult(&ctx->ecmult_ctx, &rj, &pkj, &e, &s);
 
-    return secp256k1_gej_has_quad_y_var(&rj) /* fails if rj is infinity */
```

Comment:
secp256k1_gej_has_quad_y_var  is unused after this (except for the tests). It was added in #402, presumably for the schnorr support. Should it get removed?  @sipa ?

Reply:
I don't think we have any immediate use for it in this library anymore really... not just gej_has_quad_y, but also fe_is_quad_var, num_jacobi, and possibly ge_set_xquad?

Reply:
ge_set_xquad is just the natural behaviour of using a square root to recover y.

Reply:
Yeah, that's why I said maybe. The alternative is merging it into ge_set_xo_var (which would be the only call site left, I believe).